ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/InstallPlaywrightTask.cs

```diff
@@ -0,0 +1,13 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks
+{
+    public class InstallPlaywrightTask : GithubTask
+    {
+        public override string Run { get; set; } = "dotnet tool install --global Microsoft.Playwright.CLI";
+    }
```

Comment:
Can we add code comments to all componenst so it enhances the consumers experience and also show the defaults so they can know when to override the Run value.

```suggestion
    /// <summary>
    /// A task to install Microsoft.Playwright.CLI.
    /// </summary>
    public class InstallPlaywrightTask : GithubTask
    {
        /// <summary>
        /// Gets or sets the command to execute for the task.
        /// The default value is: "dotnet tool install --global Microsoft.Playwright.CLI".
        /// </summary>
        public override string Run { get; set; } = "dotnet tool install --global Microsoft.Playwright.CLI";
    }
```